xml/ay_general_options.xml

```diff
@@ -120,7 +120,7 @@
   </calloutlist>
 
   <sect2 xml:id="CreateProfile-General-mode">
-   <title>The Mode Section</title>
+   <title>The mode section</title>
```

Comment:
Shouldn't the mode be literal? We are referring to an element of a configuration file, thus IMHO it should have a different style.